libcudacxx/include/cuda/__device/all_devices.h

```diff
@@ -21,19 +21,18 @@
 #  pragma system_header
 #endif // no system header
 
-#include <cuda/std/__cuda/api_wrapper.h>
-#include <cuda/std/cassert>
-#include <cuda/std/detail/libcxx/include/stdexcept>
-#include <cuda/std/span>
+#if _CCCL_HAS_CTK()
+#  include <cuda/__device/physical_device.h>
+#  include <cuda/std/__cuda/api_wrapper.h>
+#  include <cuda/std/cassert>
+#  include <cuda/std/detail/libcxx/include/stdexcept>
+#  include <cuda/std/span>
 
-#include <cuda/experimental/__device/device.cuh>
+#  include <vector>
```

Comment:
This will blow up the NVRTC internal header tests.

We need to make sure this is not included with NVRTC

Reply:
how large do we expect this to become?

we do have `inplace_vector`